test/parallel/test-crypto-keygen.js

```diff
@@ -1743,22 +1743,6 @@ generateKeyPair('rsa', {
   assert.strictEqual(typeof privateKey, 'string');
 }));
 
-{
-  // Proprietary Web Cryptography API ECDH/ECDSA namedCurve parameters
-  // should not be recognized in this API.
-  // See https://github.com/nodejs/node/issues/37055
```

Comment:
Technically, since this is a regression test, it's not strictly necessary to remove it.

But that's just an FYI, I don't mind removing this.